apps/docs/guides/engines/workflow-engine.mdx

```diff
@@ -441,8 +472,10 @@ await git.clone('repo', async () => {
     ```
 </ResponseField>
 
-<ResponseField name="jsFiles(glob, callback)">
-  Filters out Javascript/Typescript files.
+<ResponseField name="files(glob, callback)">
+  Filters out files using glob.
+  You can also combine this function and subcommand function:
+  Calling files().js() - will find files with js, jsx, ts, tsx, cjs, mjs extensions.
```

Comment:
jsFam @arybitskiy 
js for JavaScript only.

Reply:
@alexbit-codemod renamed js to jsFam
there is no need for separate js at this point